internal/testing/initialization/test_init.go

```diff
@@ -0,0 +1,20 @@
+// The package initializes logging and all stubs for use in tests.
+package initialization
+
+import (
+	// Initialize logging (must be kept the first)
+	_ "github.com/RHEnVision/provisioning-backend/internal/logging/testing"
+	// HTTP client stub implementations
+	_ "github.com/RHEnVision/provisioning-backend/internal/clients/image_builder/stubs"
+	_ "github.com/RHEnVision/provisioning-backend/internal/clients/sources/stubs"
+	// DAO stub implementation
+	_ "github.com/RHEnVision/provisioning-backend/internal/dao/stubs"
+	// Job queue stub
+	_ "github.com/RHEnVision/provisioning-backend/internal/jobs/stub"
+
+	"github.com/rs/zerolog/log"
+)
+
+func init() {
+	log.Logger.Debug().Msg("initialized logging, job queue for tests")
+}
```

Comment:
This should perhaps read: "initialized logging, HTTP client(s), DAO and job queue stubs"

Will fix in the next rebase.